src/topp/TextExporter.cpp

```diff
@@ -242,7 +242,14 @@ namespace OpenMS
   // stream output operator for a ProteinHit
   SVOutStream& operator<<(SVOutStream& out, const ProteinHit& hit)
   {
-    out << hit.getScore() << hit.getRank() << hit.getAccession()
+    // extract protein name from description
+    String protein_name = hit.getDescription();
+    if (protein_name.find("OS=") != std::string::npos)
```

Comment:
Cannot see an elegant way to get the new flag into the operator, or how to define multiple operators here. If there is no quick solutions, we will post-process the TextExporter output after all.

The important point of this PR is the additional protein description.